Autopilot/Path Manager/PathManager.c

```diff
@@ -22,6 +23,11 @@
 #include "../Common/UART1.h"
 #endif
 
+#define BOR_RESET (1<<1)
```

Comment:
It makes more sense to define these in StartUpErrorCodes.h, so that other things may use them. Go with a verbose name like `STARTUP_ERROR_BROWN_OUT_RESET` so that its more descriptive